src/js/twitter.js

```diff
@@ -1,3 +1,62 @@
+import { getNodesDeep } from './utils'
+
+const twitter_filter = (prev_tweet_count = 0) => {
+  const twitter_handles = []
+  const orgs = document.getElementsByClassName('org-info')
+
+  for (let i = 0; i < orgs.length; i++) {
+    const org_info = orgs[i].children
+    const children = [].slice.call(org_info)
+    const twitter_url = children.filter(
+      x => x.nodeName === 'A' && x.host === 'twitter.com'
+    )
+
+    if (twitter_url.length !== 0) {
```

Comment:
I think it's much more common to do `twitter_url.length > 0`, just because it makes it obvious that `twitter_url.length` can't be negative or a string (even though that's plainly obvious to people who know JavaScript, newcomers might not immediately know what `.length` is).